### ISSUE TITLE
`@stylistic/selector-pseudo-class-parentheses-space-inside` fix throws on a pseudo-class with an empty argument

### ISSUE THREAD

**Issue by firefoxic** [stylelint-stylistic/stylelint-stylistic#127]
Found while reviewing the fix for #123, which does not touch this and does not cause it — the behaviour is the same on either side of that commit.

### What minimal example or steps are needed to reproduce the bug?

```css
a:not() {}
```

### What minimal configuration is needed to reproduce the bug?

```json
{
  "plugins": [
    "@stylistic/stylelint-plugin"
  ],
  "rules": {
    "@stylistic/selector-pseudo-class-parentheses-space-inside": "always"
  }
}
```

### How did you run Stylelint?

```shell
stylelint "**/*.css" --config myconfig.json --fix
```

### Which Stylelint-related dependencies are you using?

```json
{
  "@stylistic/stylelint-plugin": "5.3.0",
  "stylelint": "17.14.1"
}
```

### What did you expect to happen?

Nothing to report. There is no whitespace inside those parentheses to have an opinion about, and nothing to fix.

### What actually happened?

The fixer throws, and the run ends with a parse error rather than a warning:

```
Cannot parse selector (TypeError: Cannot read properties of undefined (reading 'type'))
```

The result is marked `errored`, no warning is produced, and the file comes back unfixed. A single such selector anywhere in a stylesheet costs every fix this rule would have made in it.

Every shape with an empty argument reaches it, and the pseudo-class does not matter:

```css
a:not() {}
a:not( ) {}
a:not(,b) {}
a:not(b,) {}
a:is() {}
a:nth-child() {}
```

Under `never` none of them throws.

### Do you have a proposal to fix the bug?

An empty argument parses to a `selector` node with no children, so `pseudoNode.length` is 1 and the guard on it lets the node through. `setFirstNodeSpaceBefore` then recurses into that empty selector, `node.first` is `undefined`, and `target.type` throws. `setLastNodeSpaceAfter` has the same hole on the other end.

Only `always` reaches it: an empty selector stringifies to nothing, so `nextCharIsSpace` and `prevCharIsSpace` are both false, which is what `never` wants and what `always` tries to correct.

The guard belongs where the emptiness is: a pseudo-class none of whose arguments has a node has no inside to space out, and should be skipped alongside the `pseudoNode.length === 0` case that is already there. Both helpers should still refuse to walk into a container with no children, so that no other caller can find the same edge.
